docs/content/core/volumes.md

```diff
@@ -5,228 +5,176 @@ aliases:
-
-This is what the file sharing chain looks like with a NFS volume. 
+**On Windows with Docker Desktop** it is Docker Desktop itself that mounts all configured Windows drives via SMB.
 
 ```
-container:/var/www ==bind mount==> project_root ==> Linux:project_root ==NFS==> macOS:PROJECT_ROOT
+               SMB mount                       bind
+Windows Host =============> Docker Desktop ==============> Container   
 ```
 
-As you can see, containers mount NFS via the host machine and not directly. This setup method only makes sense on macOS 
-with Docker for Mac, for testing and performance comparison purposes.  
+To see how your project's Docker volumes are defined with `DOCKSAL_VOLUMES=bind` see 
+[stacks/volumes-bind.yml](https://github.com/docksal/docksal/blob/master/stacks/volumes-bind.yml).
 
-### Using NFS Volumes
+In most cases you do not need to set `DOCKSAL_VOLUES=bind` option. It is set for you automatically 
+in applicable cases. The only thinkable exception is when you need fsnotify events on macOS with Docker Desktop, 
+but don't want to use `unison` option.
 
-- Add `DOCKSAL_VOLUMES=nfs` either globally in `$HOME/.docksal/docksal.env` or in `.docksal/docksal.env` in a project
-- `fin project reset`
+### nfs
+
+This option is macOS specific and is used by default on macOS instead of bind. It means that containers will not wait 
+for something to make host files accessible for them first, instead they will reach out to host themselves via NFS protocol.
 
+```
+              NFS mount
+macOS Host ==============> Container  
+```
 
-## Unison Volumes
+NFS generally works faster than `osxfs` so this option is default on macOS even for Docker Desktop setup. The downside
+is that NFS does not support fsnotify events.
 
-We can also do more advanced and pretty interesting solutions, like using Unison to synchronize files between the host 
-and the `project_root` volume. 
+To see how your project's Docker volumes are defined with `DOCKSAL_VOLUMES=nfs` see 
+[stacks/volumes-nfs.yml](https://github.com/docksal/docksal/blob/master/stacks/volumes-nfs.yml). 
 
-See [stacks/volumes-unison.yml](https://github.com/docksal/docksal/blob/master/stacks/volumes-unison.yml).
+### unison
 
-Unison volumes make the most sense for Docker for Mac users as an alternative to the (still slow) `osxfs` file sharing.
+This option only works on macOS and Windows. Host files are made accessible to VM like in a bind mount, but after
+that container does not access them directly via bind to avoid NFS/SMB performance penalty. Container file system is not 
+connected to host filesystem at all.   
 
-This is what the file sharing chain looks like with Unison over `osxfs`. 
+Instead `fin` launches a `unison` container with Unison daemon that copies files back and forth between mounted host 
+files and the container. Transferrring changes from host to the container and back becomes slower, but reading and writing 
+the files within the container becomes way faster.
```

Comment:
becomes much faster.

Reply:
what's wrong with "way faster"?

Reply:
There is nothing “wrong” except that it’s more slang. “Much” would just be the preferred.